sympy/geometry/ellipse.py

```diff
@@ -543,6 +543,36 @@ def focus_distance(self):
         """
         return Point.distance(self.center, self.foci[0])
 
+    def auxiliary_circle(self, x='x', y='y'):
+        """The equation of auxiliary circle of the ellipse.
+
+        Returns
+        =======
+
+        equation : sympy expression
+
+        Parameters
+        ==========
+
+        x : str, optional Label for the x-axis. Default value is 'x'.
+        y : str, optional Label for the y-axis. Default value is 'y'.
+
+        Examples
+        ========
+
+        >>> from sympy import Ellipse, Point
+        >>> ellipse = Ellipse(Point(2, 4), 9, 1)
+        >>> ellipse.auxiliary_circle()
+        (x - 2)**2 + (y - 4)**2 - 81
+
+        """
+        x = _symbol(x)
+        y = _symbol(y)
+        m =_symbol(m)
+        hr, vr = self.hradius, self.vradius
+        m = max(hr, vr)
```

Comment:
If `hr` and `vr` are symbols, there will likely be a symbolic comparison in `max`.

Reply:
`Max` could be used here.